src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/MercuryBasePageObject.java

```diff
@@ -0,0 +1,44 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.mercury;
+
+import com.wikia.webdriver.common.contentpatterns.URLsContent;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+
+import org.joda.time.DateTime;
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class MercuryBasePageObject extends BasePageObject {
+
+  public MercuryBasePageObject(WebDriver driver) {
+    super(driver);
+  }
+
+  public CuratedMainPagePageObject openCuratedMainPage(String wikiURL, String mainPage) {
+    getUrl(wikiURL + URLsContent.WIKI_DIR + mainPage + "?cb=" + DateTime.now().getMillis());
```

Comment:
you can use urlBuilder to create this url.
It works nicely with appending query params